src/App.css

```diff
@@ -1,39 +1,43 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.contianer {
+  width: 40%;
+  height: auto;
+  border: 2px solid #000;
+  border-radius: 5px;
```

Comment:
- [ ] As requested by the project requirement, your design should match exactly the design provided. Kindly, remove the black border to your container to fit the design provided.

Reply:
Thank you for the code review, I have removed the black border.